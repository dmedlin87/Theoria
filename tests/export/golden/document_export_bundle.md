---
title: Theo Export export-fixture
schema_version: 2024-07-01
created_at: 2024-01-01T00:00:00+00:00
export_type: documents
record_count: 1
---

## Manifest
```json
{
  "export_id": "export-fixture",
  "schema_version": "2024-07-01",
  "created_at": "2024-01-01T00:00:00Z",
  "type": "documents",
  "filters": {
    "collection": "theology",
    "author": null,
    "source_type": null
  },
  "totals": {
    "documents": 1,
    "passages": 1,
    "returned": 1
  },
  "app_git_sha": "abc1234",
  "enrichment_version": 2,
  "cursor": "doc-1",
  "next_cursor": null,
  "mode": null
}
```

## Records
```json
[
  {
    "kind": "document",
    "document_id": "doc-1",
    "title": "Example Document",
    "collection": "theology",
    "source_type": "article",
    "authors": [
      "Jane Doe",
      "John Roe"
    ],
    "doi": "10.1234/example",
    "venue": "TheoConf",
    "year": 2023,
    "topics": [
      "grace"
    ],
    "primary_topic": "Grace",
    "enrichment_version": 2,
    "provenance_score": 82,
    "abstract": "An example abstract",
    "source_url": "https://example.com/doc-1",
    "metadata": {
      "publisher": "Theo Press",
      "pages": "10-12"
    },
    "passages": [
      {
        "id": "passage-1",
        "document_id": "doc-1",
        "osis_ref": "John.1.1",
        "page_no": 1,
        "t_start": null,
        "t_end": null,
        "text": "In the beginning was the Word.",
        "meta": {
          "snippet": "In the beginning was the Word."
        }
      }
    ]
  }
]
```

%% Theo export provenance manifest embedded above %%
