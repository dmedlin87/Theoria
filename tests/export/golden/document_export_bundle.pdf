%PDF-1.4
%
1 0 obj
<< /Type /Catalog /Pages 2 0 R >>
endobj
2 0 obj
<< /Type /Pages /Kids [3 0 R] /Count 1 >>
endobj
3 0 obj
<< /Type /Page /Parent 2 0 R /MediaBox [0 0 612 792] /Contents 4 0 R /Resources << /Font << /F1 5 0 R >> >> >>
endobj
4 0 obj
<< /Length 2205 >>
stream
BT
/F1 11 Tf
14 TL
72 760 Td
(Theo Export export-fixture) Tj
T*
(Schema Version: 2024-07-01) Tj
T*
(Created: 2024-01-01T00:00:00+00:00) Tj
T*
(Export Type: documents) Tj
T*
() Tj
T*
(Manifest:) Tj
T*
({) Tj
T*
(  "export_id": "export-fixture",) Tj
T*
(  "schema_version": "2024-07-01",) Tj
T*
(  "created_at": "2024-01-01T00:00:00Z",) Tj
T*
(  "type": "documents",) Tj
T*
(  "filters": {) Tj
T*
(    "collection": "theology",) Tj
T*
(    "author": null,) Tj
T*
(    "source_type": null) Tj
T*
(  },) Tj
T*
(  "totals": {) Tj
T*
(    "documents": 1,) Tj
T*
(    "passages": 1,) Tj
T*
(    "returned": 1) Tj
T*
(  },) Tj
T*
(  "app_git_sha": "abc1234",) Tj
T*
(  "enrichment_version": 2,) Tj
T*
(  "cursor": "doc-1",) Tj
T*
(  "next_cursor": null,) Tj
T*
(  "mode": null) Tj
T*
(}) Tj
T*
() Tj
T*
(Records:) Tj
T*
([) Tj
T*
(  {) Tj
T*
(    "kind": "document",) Tj
T*
(    "document_id": "doc-1",) Tj
T*
(    "title": "Example Document",) Tj
T*
(    "collection": "theology",) Tj
T*
(    "source_type": "article",) Tj
T*
(    "authors": [) Tj
T*
(      "Jane Doe",) Tj
T*
(      "John Roe") Tj
T*
(    ],) Tj
T*
(    "doi": "10.1234/example",) Tj
T*
(    "venue": "TheoConf",) Tj
T*
(    "year": 2023,) Tj
T*
(    "topics": [) Tj
T*
(      "grace") Tj
T*
(    ],) Tj
T*
(    "primary_topic": "Grace",) Tj
T*
(    "enrichment_version": 2,) Tj
T*
(    "provenance_score": 82,) Tj
T*
(    "abstract": "An example abstract",) Tj
T*
(    "source_url": "https://example.com/doc-1",) Tj
T*
(    "metadata": {) Tj
T*
(      "publisher": "Theo Press",) Tj
T*
(      "pages": "10-12") Tj
T*
(    },) Tj
T*
(    "passages": [) Tj
T*
(      {) Tj
T*
(        "id": "passage-1",) Tj
T*
(        "document_id": "doc-1",) Tj
T*
(        "osis_ref": "John.1.1",) Tj
T*
(        "page_no": 1,) Tj
T*
(        "t_start": null,) Tj
T*
(        "t_end": null,) Tj
T*
(        "text": "In the beginning was the Word.",) Tj
T*
(        "meta": {) Tj
T*
(          "snippet": "In the beginning was the Word.") Tj
T*
(        }) Tj
T*
(      }) Tj
T*
(    ]) Tj
T*
(  }) Tj
T*
(]) Tj
ET
endstream
endobj
5 0 obj
<< /Type /Font /Subtype /Type1 /BaseFont /Helvetica >>
endobj
6 0 obj
<< /Producer (Theo Exporter) /Creator (Theo Exporter) /Title (Theo Export export-fixture) /Subject (Schema 2024-07-01) /CreationDate (D:20240101000000+0000 )>>
endobj
xref
0 7
0000000000 65535 f 
0000000015 00000 n 
0000000067 00000 n 
0000000127 00000 n 
0000000256 00000 n 
0000002519 00000 n 
0000002592 00000 n 
trailer
<< /Size 7 /Root 1 0 R /Info 6 0 R >>
startxref
2770
%%EOF
