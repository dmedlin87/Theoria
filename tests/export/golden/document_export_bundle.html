<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <title>Theo Export export-fixture</title>
    <meta name="generator" content="Theo Exporter" />
    <meta name="theo:schema_version" content="2024-07-01" />
    <script id="theo-export-manifest" type="application/json">{
  "export_id": "export-fixture",
  "schema_version": "2024-07-01",
  "created_at": "2024-01-01T00:00:00Z",
  "type": "documents",
  "filters": {
    "collection": "theology",
    "author": null,
    "source_type": null
  },
  "totals": {
    "documents": 1,
    "passages": 1,
    "returned": 1
  },
  "app_git_sha": "abc1234",
  "enrichment_version": 2,
  "cursor": "doc-1",
  "next_cursor": null,
  "mode": null
}</script>
    <script id="theo-export-records" type="application/json">[
  {
    "kind": "document",
    "document_id": "doc-1",
    "title": "Example Document",
    "collection": "theology",
    "source_type": "article",
    "authors": [
      "Jane Doe",
      "John Roe"
    ],
    "doi": "10.1234/example",
    "venue": "TheoConf",
    "year": 2023,
    "topics": [
      "grace"
    ],
    "primary_topic": "Grace",
    "enrichment_version": 2,
    "provenance_score": 82,
    "abstract": "An example abstract",
    "source_url": "https://example.com/doc-1",
    "metadata": {
      "publisher": "Theo Press",
      "pages": "10-12"
    },
    "passages": [
      {
        "id": "passage-1",
        "document_id": "doc-1",
        "osis_ref": "John.1.1",
        "page_no": 1,
        "t_start": null,
        "t_end": null,
        "text": "In the beginning was the Word.",
        "meta": {
          "snippet": "In the beginning was the Word."
        }
      }
    ]
  }
]</script>
    <style>
      body {
        font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
        margin: 2rem;
        line-height: 1.5;
      }
      pre {
        background: #f5f5f5;
        border-radius: 0.5rem;
        padding: 1rem;
        overflow-x: auto;
      }
      h1, h2 {
        color: #1a365d;
      }
      .summary {
        margin-bottom: 2rem;
      }
      .summary dt {
        font-weight: 600;
      }
    </style>
  </head>
  <body>
    <main>
      <header>
        <h1>Theo Export export-fixture</h1>
        <dl class="summary">
          <dt>Schema version</dt>
          <dd>2024-07-01</dd>
          <dt>Created</dt>
          <dd>2024-01-01T00:00:00+00:00</dd>
          <dt>Export type</dt>
          <dd>documents</dd>
        </dl>
      </header>
      <section id="manifest">
        <h2>Manifest</h2>
        <pre>{
  &quot;export_id&quot;: &quot;export-fixture&quot;,
  &quot;schema_version&quot;: &quot;2024-07-01&quot;,
  &quot;created_at&quot;: &quot;2024-01-01T00:00:00Z&quot;,
  &quot;type&quot;: &quot;documents&quot;,
  &quot;filters&quot;: {
    &quot;collection&quot;: &quot;theology&quot;,
    &quot;author&quot;: null,
    &quot;source_type&quot;: null
  },
  &quot;totals&quot;: {
    &quot;documents&quot;: 1,
    &quot;passages&quot;: 1,
    &quot;returned&quot;: 1
  },
  &quot;app_git_sha&quot;: &quot;abc1234&quot;,
  &quot;enrichment_version&quot;: 2,
  &quot;cursor&quot;: &quot;doc-1&quot;,
  &quot;next_cursor&quot;: null,
  &quot;mode&quot;: null
}</pre>
      </section>
      <section id="records">
        <h2>Records</h2>
        <pre>[
  {
    &quot;kind&quot;: &quot;document&quot;,
    &quot;document_id&quot;: &quot;doc-1&quot;,
    &quot;title&quot;: &quot;Example Document&quot;,
    &quot;collection&quot;: &quot;theology&quot;,
    &quot;source_type&quot;: &quot;article&quot;,
    &quot;authors&quot;: [
      &quot;Jane Doe&quot;,
      &quot;John Roe&quot;
    ],
    &quot;doi&quot;: &quot;10.1234/example&quot;,
    &quot;venue&quot;: &quot;TheoConf&quot;,
    &quot;year&quot;: 2023,
    &quot;topics&quot;: [
      &quot;grace&quot;
    ],
    &quot;primary_topic&quot;: &quot;Grace&quot;,
    &quot;enrichment_version&quot;: 2,
    &quot;provenance_score&quot;: 82,
    &quot;abstract&quot;: &quot;An example abstract&quot;,
    &quot;source_url&quot;: &quot;https://example.com/doc-1&quot;,
    &quot;metadata&quot;: {
      &quot;publisher&quot;: &quot;Theo Press&quot;,
      &quot;pages&quot;: &quot;10-12&quot;
    },
    &quot;passages&quot;: [
      {
        &quot;id&quot;: &quot;passage-1&quot;,
        &quot;document_id&quot;: &quot;doc-1&quot;,
        &quot;osis_ref&quot;: &quot;John.1.1&quot;,
        &quot;page_no&quot;: 1,
        &quot;t_start&quot;: null,
        &quot;t_end&quot;: null,
        &quot;text&quot;: &quot;In the beginning was the Word.&quot;,
        &quot;meta&quot;: {
          &quot;snippet&quot;: &quot;In the beginning was the Word.&quot;
        }
      }
    ]
  }
]</pre>
      </section>
    </main>
  </body>
</html>
